superduperdb/components/model.py

```diff
@@ -524,10 +608,25 @@ def __post_init__(self, artifacts):
         if self.model_to_device_method is not None:
             self._artifact_method = getattr(self, self.model_to_device_method)
 
+    def __repr__(self):
+        s = f'Identifier: {self.identifier}\nParams: {self.inputs.params}\nPreferred Devices: {self.preferred_devices}'
+        return border_msg(s, title='SuperDuper Model')
+
     def to_call(self, X, *args, **kwargs):
+        if isinstance(X, (tuple, list)):
```

Comment:
How about X is exactly a list as one parameter?
For example: 
1. Batch inference `model.predict(["1", "2"])`
2. Text similarity `model.predict(("This is a apple", "There are two apple"))`

Reply:
This should be handled explicitly with the flag `one=True/ False`. There is no general way to know if we are in batched or single data point mode.